### DISCUSSION TITLE
Will steps be taken to increase availability?

### DISCUSSION THREAD

**Discussion by rockztah** [frntc/RAD#9] [General]
Hi!

The RAD Expansion Unit looks great and I would love to get my hands on one.

Availability is low at the moment, however. Will you be allowing any other webshops besides https://restore-store.de to sell it in order to increase availability? Retro-restore are currently not accepting orders until march 27th. Also, it would seem the 3A+/3B+ version is not in stock at the moment. Besides, products from Restore-store don't come fully assembled, which is a major hurdle for someone like me, who is not very handy with a soldering iron. I still have my Sidekick64 that I received some two years ago lying unassembled in a drawer.

**Comment by COREi64** [comment]
The COREi64.com shop would be happy to provide them to the community, assembled and tested, even with cases that have been designed by COREi64.com.  I have entered into agreements with many different developers over the years with great success and have developed  a solid base of support in our community for quality products for fair prices.  I have the skills to produce professionally built boards for a fair market price.  Just need the "go-ahead" and availability could be increased.  Say the word Frenetic, and we can be off and running.  Currently however, due to license restrictions, they are not available through COREi64.com.  I had been offering them briefly through lemon64 as a public service to those in the forum without the skills, however, due to objections, I've had to rescind the offer to the community.

If Frenetic is amenable to my offer, availability COULD be increased.

**Comment by doughamm** [comment]
As a Canadian retro computer user I find it incredibly expensive to order from shops in Europe - something anyone in North or South America would attest to. Corei64 graciously built my RAD since I lack the skills and tools to do it properly -- and it's arguably the most professional build in my hardware collection.  

This device and the Sidekick64 are simply too cool and too important to the preservation of the Commodore spirit of innovation to be allowed to fade into obscurity because of their complexity in building. They're also too...persnickety?... to produce using inferior methods.  I have no selfish interest in hoarding these things as rarities - they should be widely available and enable a new generation of incredible Commodore games and apps. Please, please extend the rights for producing them to Corei64!

**Comment by paulhocker** [comment]
I agree with @doughamm. I was thrilled that I was able to get a RAD and SK from Thomas. His build quality is excellent and the price was very, very reasonable. Plus I did not have to wait three weeks for delivery from the UK. I can do some basic soldering, but those smaller on board components are out of my league. I appreciate someone being able to do it for me. I believe a lot of people would really enjoy these wonderful devices. Although, finding a RasPI these days can be difficult.

**Comment by jkadaver** [comment]
These are really good products that definitely need to see a wider base of distribution and collaboration with good and talented people I feel this is essential. People who will care about the end product & the quality of their work. Apparently there’s some sellers trying to sell these as half baked kits. Where only the SMD parts are installed. The problem with this is a lot of people don’t have the skills or tools necessary to finish these and you’re leaving this open for failure, where it could easily be just shifted off onto the consumer, when you could easily make partnerships with trusted and verified makers that care about the quality & the end product. These products are something, that’s definitely worth keeping alive as trying to find. For example, an OEM ram expansion unit on eBay for example will easily make the strongest man’s eyes water like cutting an onion. The problem is with these type of products are that the communities that care about these types of products are very small, so the more people that work together to make these kind of things available for the highest quality possible is essential and especially if affordable. When things like this are hard to come bye or become overly expensive due to shipping cost especially they die off because nobody can afford to obtain them so please work with other makers to make these as readily available as possible. 
I live in North America and I can attest to the quality of the end product from COREi64.com you get what you pay for in quality, gold connectors everything looks plum and professional for affordable and reasonable prices. He also a seller that’s easy to communicate with and very open about the process I mean Thomas made a YouTube video building one of the RAD units. So the visual quality alone shouldn’t be a question you can see for yourself..

**Comment by RetroBadgeman** [comment]
Ordering a Pi was quite a hassle but I really wanted a SideKick64. Because I lack the ability to solder it myself I reached out to a respectable member of the community, Thomas from COREI64.com. Not only does he make perfectly assembled boards and ditto cases, he also does this time consuming job for a very fair price. So extending the rights and allowing him to be able to sell this wonderful cartridge to more people in the retro community who lack the skill to make one themselves would be awesome!

**Comment by LeviJSmith** [comment]
I agree with the others.  Thomas from COREi64.com does great work and offers tech at a fair price.  I would love to see you extend the rights to him to be able to sell the RAD & SideKick cartridges.

**Comment by Marc91360** [comment]
As RetroBadgeman says, finding a RB Pi is by itself a real pain. But I can understand why.
What I don't understand is why it's virtually impossible to find a RAD or a SideKick64. 

I have no skill in soldering. 

Why not allowing a small number of people, known and recognized members of the community - I'm thinking about Thomas from COREi64.com - to sell these little wonders ? 
What exactly does Frenetic expect from us, and what are its plans for these cartridges, if he prevents them from being put on the market? If at least he sold them by himself ... I admit I don't quite understand.
Best regards from France

**Comment by frntc** [comment]
Great, that you like my projects! It was fun creating them, and believe me, it was hundreds or thousands of hours that were needed to get to this point.

@rockztah @Marc91360 restore-store will be online again on Monday, I’m sure if you ask Konstantin, he will help with soldering the buttons and the connector. Dennis from retro-updates sells fully assembled Sidekick64s. If there are sold out, just ask him when he has then back again!

Moreover, I’m currently talking to two other potential sellers (US and EU). 

Let me also point out that I did not only put spare time into the development of the software and hardware, I also updated the PCB designs several times to avoid IC supply bottlenecks. Last time just a couple of weeks ago: https://www.forum64.de/index.php?thread/52749-heute-so-gebastelt/&postID=1971993#post1971993

**Reply by doughamm** [reply]
This is great to hear!  I would still urge you to send some love to Canada by accommodating Corei64, as he's based in our country.  Paying even half the price in USA funds would result in shipping and import taxes to Canada that will net a price increase over what Thomas has been charging.  Furthermore, a little wider distribution would ensure sufficient competition to keep quality high and prices appropriate.

Just my $.02.  :)

**Reply by Marc91360** [reply]
Alas ! Both Restore-store and Retro-updates are out of stock - and this for some time right now.

**Reply by frntc** [reply]
restore-store has just received 60x Sidekick64 out of professional SMD assembly lines.

**Comment by mstine** [comment]
To pile on... I have a Corei64 RAD. You know that from https://github.com/frntc/RAD/discussions/8. Not only is it well manufactured, but Thomas is just an all-around good person and wants to help the community on this side of the pond. When I thought my board was blown, he offered to repair it without being asked. He's not trying to get rich by producing your designed products. Having had experience with plenty of sellers that price gouge, I know that's not what he does. 

It's rather bizarre to me that given the amount of support Thomas has from the community, your only response says, "I'm talking to two other potential sellers." It's your design, and obviously, you have the right to do anything with it. But withholding the right to sell from someone that builds a high-quality product at a very reasonable price for parts and their contributed craftsmanship makes little sense to me.

**Reply by tazmandevil2008** [reply]
Just to chime in  I cannot solder unless torching boards to a crisp is called soldering.  I hope you would allow COREi64.com shop to produce the sidekick 64 and the Rad board too.  Over the years I have purchased his products they are first rate,  Prices are fair for the soldering and assembly.  They are almost works of art ,the quality is first rate and he tests every board before shipping. He has made products that were almost lost but made available to thee 8 bit community.

**Comment by COREi64** [comment]
It seems that Frenetic has chosen not to address my application for commercial status here, but rather, reply to me through a PM in Lemon64.  It has been made clear to me that he has no intention of offering commercial status.  This is entirely his right.

As a result, I will resume offering his creations as per the description in his github page that states "...It is also perfectly fine and appreciated if someone sells spare PCBs (populated or not) of a PCB-order or manufactures a small batch and offers them on a forum..."

As stated in the github post, I am sure that Frenetic will appreciate this effort.  Rest assured however, you will NOT see these commercially available through my shop as Frenetic has indicated through PM no interest in making such arrangements with me.  Why is that?  I honestly have no idea at all, as I never breached the non-commercial license, as they were NEVER available through my shop.

For those seeking additional availability options, find me on the community forums, and I'll happily set you up with a board so that you may enjoy it.

And Frenetic, no hard feelings, I'm a huge fan of your creations and am pleased that you appreciate the efforts of those making them for the community.

To those that have rallied to try to achieve a commercial status, thank-you for your support!  It's really terrific to see such passionate individuals.  I feel honored to be in your company.  All the best.

PS.  Just to clarify any confusion, when people say they have a "COREi64 RAD" or a "COREi64 Sidekick64" they are referring to my forum user id, not my shop!

**Comment by hpingel** [comment]
Just for the sake of some balance in this discussion I would like to add that I am one of Frenetic's (@frntc) testers and have been closely following his projects since 2019 (maybe even earlier?). I have spent a huge part of my spare time over the last three years testing Frenetic's projects based on my curioistity on a voluntary basis.

I have more thoughts on this topic but for now I just want to add this:

I can fully understand Frenetic being careful and hesitant about choosing partners for selling his projects commercially which is as such ruled out by the chosen licence. And I can see nothing wrong in his behaviour so far in this matter.

For me it feels like the purpose of this discussion is to build up some pressure for Frenetic to make the "right" decision towards global sales which is inapproriate for a Github discussion and inappropriate in many other senses in my opinion. 

At the same time I can also understand the frustration of you who simply want to get a RAD/Sidekick64 and can't get your hands on one. The frustration is also partly due to the fact that Raspberry Pi's are extremely hard to obtain these days due to the global chip crisis (that was already announced to be over and done with). And the chip crisis still affects the availability of the chips on the Sidekick PCB at this point in time.

One more thought: The RAD project was released to the public in late November 2022. So it has only been out to the world for four months. Where does the urgency come from to suddenly make this available commercially when Sidekick64 had much more time to grow without this sudden eagerness to sell it globally? Is it that the RAD promises a commercial success like Pi1541 had as of 2018?

**Reply by Marc91360** [reply]
Hello Hpingel,

I can speak only for myself; it's more the SideKick64 that I am looking for, and I already have the RB Pi part. So I'm not really concerned with the arguments made here.
My only question is why invent such a beautiful thing, then do everything to prevent it from knowing the success it deserves?
My friends and me are a small group of French enthusiasts, and several of us are looking for this cartridge!

The result is that it's far more easy to find a Kung Fu Flash (in my opinion much less interesting) than a SideKick64 ... The KKF is taking all the light !

What are the motivations and the projects of Master Frenetic for the community which has only one desire - to pay him the homage he deserves by using his creations World-Wide ?

Best regards,

Marc

**Comment by rockztah** [comment]
Original poster here. Well, this kind of blew up in my face...

@hpingel: No, I did NOT start this discussion in order to "build pressure", as you imply. It was NOT a battle cry for COREi64 or any specific seller. 

I am Europe based, and just wish raspberry-based stuff for the C64 to be available to anyone, as is the case on the Amiga scene. I was genuinely curious as to what was being done to to make it easier to get hold of the RAD Expansion Unit, having just been blown off by an ebay seller who cancelled my order because I lived in the wrong country, and having visited restore-store.de only to be told that the product was unavailable. 

Had I known the kind of backlash my question would inspire, I would have refrained from broaching the subject in an open forum. I was completely ignorant of the explosive power it contained. I originally wanted to pose this question Frenetic directly, but could find no way of doing so through github.

@frntc : Thank you so much for taking the time to answer. It's reassuring to learn that you are talking to other sellers.

It was never my intention to belittle your efforts at developing and disseminating your projects. I am well aware that a tremendous amount of time and effort has been poured into these projects of yours, and the c64 community has a lot to thank you for.

Its great to finally see some rpi-based peripherals emerging for the C64, and it would be a crying shame if they were to fade into obscurity because no one could lay their hands on them. I am a huge fan of the Pistorm boards for the Amiga, they are open source and available in most Amiga shops. That's what I want to see for your wonderful projects. 

I assure you, no offence was intended, and I am truly sorry if what I thought was as an innocent question has caused you grief.

**Reply by hpingel** [reply]
Thank you for your explanation of your motivation, I understand and I see your point. Due to limited time this answer is as short as is.

**Comment by frntc** [comment]
Dear all,

this situation is becoming frustrating, it should not be one in which I feel like defending myself. Your compliments are a motivation to continue with such projects, exerting pressure (here or on FB) and license violations have the opposite effect.

I proactively approached COREi64 to resolve the situation, but over the course of our private communication I felt there’s unfortunately no common denominator to proceed at this point in time. I’ve invested a lot of time in my open source projects (and also @emulaThor, thanks to him for testing and lots of valuable feedback during development!) and I’m happy to hear from people enjoying them. But please, respect my decisions, I’m sure there will always be ways to get your hands on a Sidekick64 or RAD. If you have questions, feel free to reach out to me via mail or PM on forum64.de (preferred) or lemon64.com

My mail address can be found in the header of every source file, e.g. https://github.com/frntc/RAD/blob/main/Source/Firmware/rad_main.cpp

**Reply by COREi64** [reply]
Frenetic, lets be honest here.  Your statement of "...I proactively approached COREi64 to resolve the situation..." couldn't be further from the truth now is it.  I sent you a message a couple weeks ago asking how we may move forward and possibly with an agreement, and all I got was crickets.  You ignoring my request does not constitute you proactively doing anything.  I asked what the terms of a licensing agreement would represent, and you never replied to me.  And just recently you simply declined any opportunity of the aforementioned collaboration to assist in the community.

You don't want to deal with me, hey that's fine, but don't paint a false picture of your attempts to work things out with me through PM's.

**Reply by frntc** [reply]
Please take a look at our conversation again. I started it on February 22 and explained to you on March 9 why I came to the conclusion to not proceed with an agreement.

**Comment by doughamm** [comment]
I think what I've read above from both gents is a great compromise -- thank you both for being mature and responsible in this public forum.  :)  There's evidently more here than what us ignorant pundits understand, and all the better that it stay that way.

@hpingel, you asked great questions and I assume they weren't rhetorical, so I'm happy to share my motivations.  For starters, I own both devices thanks to Thomas having provided them through a private forum interaction -- I'm set for life.  :)  My concerns are primarily that:

- Living in Canada, shipping/duty/exchange rate from anywhere but Canada is prohibitively expensive., yet we have a thriving retro scene here (Thomas, Robin Harbron, and other YouTubers are just the tip of the iceberg).  When I see a Canadian produce high quality output, I'll eagerly support them.
- I run a small FaceBook group in Manitoba and there are several C64 enthusiasts who lack the skills and funds to enjoy a product like this without the above costs and options being accessible.  I host a user group, it gets demoed, and then people are frustrated at the barrier of entry.  (Pi's are bad enough!)
- More broadly speaking, we need more interest and investment in NTSC-ville - Canada, USA and elsewhere.  Nuvies are PAL only.  The latest Doom beta is, at least from what I gleaned, 50Hz (though I haven't yet tested what that means on an NTSC machine).  Any opportunity to expand the exposure and interest in ways that promote development of NTSC software on these AWESOME devices is worth my support.
- Somewhat related, there is amazing software released that requires an REU to perform well on NTSC machines at 60fps, and I want that software to be as accessible as possible.  For that we need products like the RAD to take off.
- And yet more broadly -- I see these devices, like the pi1541, BackBit Pro, and FujiNet as ensuring the survival of the retro scene worldwide.  That's worth being enthusiastic about, and it's worth (in my opinion) a bit of creative tension as seen in this thread.  At the end of the day, what I read above is NOT opposing viewpoints -- only slightly tangential -- and toward a common goal.

Cheers!

**Comment by mstine** [comment]
I have a few words to add before considering the matter closed.

I respect your decision to choose who uses your creations and how they use them, at least insomuch as a choice is possible. As a fellow creator, I value the same. I feel like I walked into the middle of drama I didn’t expect to find in the retro computing community.

I’m primarily a software engineer, and the C64 projects I’m building right now require an REU. To acquire an original Commodore or CMD REU that is confirmed working is extremely rare, and even if you find one, you’re likely to spend approaching $500 USD. On a 40-year old piece of hardware.

For me, the hardware is a tool to produce software that works. I need reliability and availability. A modern solution for the REU is the only thing that makes sense. I don’t mind overpaying for a Raspberry Pi to deal with limited supply. I bought two. I fully intend to learn soldering to reduce my dependence on external hardware production, but like most of us, this is a hobby. I spend my limited time cranking out 6502 and BASIC, not assembling boards. I am the kind of person who wants to pay someone to build me 2-3 of these things in case I burn one out in the middle of a software release.

Right now, that kind of supply chain isn’t available to me. People like me who are willing to pay whatever a fair market price for parts and labor happens to be on the RAD are shut out by these kinds of controls. I’m not enough of a hobbyist to build it myself or wait patiently for 1-2 sellers to get around to having stock. You have the best solution design available, but if my board blows tomorrow, my project goes on hold. Not right or wrong, just the facts.

And that’s, again, obviously your decision that you’ve asked us to respect. What makes it frustrating is the lack of transparency around your decision making. You’ve placed your criteria for what constitutes breach of license. You’ve also asked us to complain to sellers who we see violating your license, asking them “why they are not respecting open source/CC developers' licenses.”

When it comes to Thomas (COREi64)’s situation, you’ve chosen to tell us he won’t be allowed to sell the boards commercially, but not why. As far as I can tell, everyone that’s done business with him has had a positive experience, and he has acted in good faith. So it’s surprising that y’all can’t come together. And that leads me to believe that I’m missing the context I need to understand the decision. Oh well, this ship seems to have sailed.

There’s one last matter. It’s also clearly stated in the README that informal sales through forums is totally fine. That’s ultimately how everyone that’s worked with Thomas got a board anyway. He’s made a clear public statement of his intention to continue these sales. I think you should make a clear public statement as to whether or not you consider these sales a compliant example of informal forum sales. This does two things:

1. It resolves the ethical tension some of us may be feeling. As I said, I walked into the middle of this by surprise. A colleague recommended I search for the RAD on eBay. That was the first I’d heard of it. I found one that was reasonably priced and bought it. I only saw the bits about authorized sellers when I later went to image my SD Card. Oops! Later, when I was troubleshooting what I thought was a fried RAD, I accidentally connected the dots to Thomas, and here we are. Now I feel like one more person that’s poured gasoline on this fire. And beyond that, I do want to buy more boards from Thomas informally, but I don’t want my conscience nagging me every time I do it.

2. It makes it a little more clear to other folks that want to help the community by offering to build small batches of RADs what is compliant. If I were in this category, and I read this thread, my first reaction would be “NOPE!” Because it’s absolutely not clear what behavior is going to get you into trouble. If it were, I doubt this thread would have gotten off the ground.

Anyway, apologies for the novel. I just feel like there’s more perspectives to any product’s ecosystem than just designer and manufacturer. Especially once you decide to share it with the world and people start to depend on it.

Cheers, and thanks again all of you for everything you do.

**Comment by frntc** [comment]
Important: First of all, I do not blame anyone who bought a Sidekick64 or RAD. On the contrary! There is absolutely no „ethical tension“ for you.

Second, as already stated on github, it is also perfectly fine and appreciated if someone sells surplus PCBs of a PCB-order or if someone manufactures a small batch for friends, retro club members, ... A small batch is what you typically order from PCB companies, 5 or 10 units. 

Note that recently there has been a batch of RADs on Forum64 of more than 100 units – and I fully appreciated that because this was a real service to the community: they have been given away for 9 Euros per piece! (maybe you guys should joint forum64.de to not miss activities like this! :-))

In general, what I expect is that there is no commercial intent (unless explicitly permitted), which I think can be seen from the price tag. If a Sidekick64 or RAD is equally priced or more pricey than one from the official sellers (who have to be conformal with all the regulations of packaging, pay taxes etc.) then … well. 

(@mstine as said above, reach out to me)

**Reply by COREi64** [reply]
Frenetic.  Just to counter a couple of your statements here for clarity, you've never been clear as to what a "small batch" represents.  One persons interpretation is different than another.  For me, 5-10 would be what I'd do for myself and certain isn't what I'd call a "batch".

Secondly, isn't that awesome that someone built 100 units and literally gave them away for 9 Euros each.  Here in my country, you couldn't buy the parts for 9 Euros.  To expect people in different economic climates to be able to match up is unrealistic.  I'd seriously like to see what 9 euros gets you.  I can't imagine you're getting an Enig board or any kind of professional quality or tested item.  I could be wrong, but in my personal experience, you get what you pay for.  Add tracked insured air shipping to that 9 euros for what is likely an untested board, you're probably looking at minimally 40 USD for those on this side of the pond for a board you still have to finish assembling more than likely.  So, for clarity, 9 Euros is certainly not what the end user pays now is it.

**Reply by hpingel** [reply]
This is the thread where the RAD was offered at that low price on ENIG:
https://www.forum64.de/index.php?thread/132530-rad-expansion-unit-bausatz-sammelbestellung/

**Reply by COREi64** [reply]
Great, thanks for posting the link.  So I was pretty much on point with my evaluation, only off on the enig part.  My German is pretty rusty, but it seems that for 9 Euros, you get an untested kit.  SMD parts are applied in some fashion, but the rest is up to the buyer to solder...  AND, no shipping outside of the EU is stated.  So ya, bottom line, it's not 9 Euros for anyone and for those outside of the EU it'll be way north when tracked insured air shipping is piled on.

**Reply by hpingel** [reply]
I solder RADs for my own use and a RAD may be tested without soldering the 2x20pin header by just holding the Pi in position with the GPIOs in the holes. This is not rocket science. Besides that the kind person who offered the pcbs would then send out additional kits to those who thought they had problems. The people offering these services are top notch so please do no wrong assumptions about other peoples skills after reading into a thread for 30 seconds.

**Reply by frntc** [reply]
I have actually two of those myself and they are of great quality. I haven't heard of anyone experiencing problems with them. 

And this was an example of service for the community which I appreciated although it was not a small batch. The upper limit of what I consider non-commercial I stated above.

**Comment by mstine** [comment]
You are both missing the point.

I want a plug and play device. I and plenty of others are willing to pay market price.

No one but Thomas seems to be willing to make a plug and play device and keep it in stock. He wants to cover his costs. But the license prohibits him from providing us with the service we are more than willing to pay him to provide.

So, instead, we wait. Because while you’re right, soldering isn’t rocket science, I have never soldered before. And I own no equipment. So if I didn’t have a RAD already, it would definitely cost me far more to build my own than to source one elsewhere. And there is no elsewhere at the moment. And six months from now if I burn out another board, there’s no guarantee of license compliant available stock.

What really blows my mind about this is that the dispute is over roughly $15 USD sticker price between the restore-store.de retail price and the price I paid Thomas. I can’t buy a decent pour of scotch in NYC for that price. No one is getting rich from it.

And if that’s not the dispute, please do educate us. I continue to find this entire situation profoundly confusing.

**Reply by frntc** [reply]
The dispute it not about the price at all. Everything I said about pricing refers to non-commercial sellers.

**Comment by frntc** [comment]
Dear all,

I will close this thread now. All is said, there won't be any new insights.

**Comment by COREi64** [comment]
Ok super.  Now we're starting to get somewhere.  100 units is as you state "...The upper limit of what I consider non-commercial...".  I have sold nowhere near that number, so we're good.  Now, for my interpretation of price below commercial sellers...  

Restore-store
$16.55 euro for a KIT plus shipping and handling for the RAD
$19.66 euro for a KIT plus shipping and handling for the Sidekick64

And your other commercial EU shop:
Retro-updates.com
$34.95 euro for what looks like a completed unit for the Sidekick64.

As your github states non-commercial small batch sellers are "expected" to sell for less, then if I charge less than say the Retro-updates site for example for a small batch run (of 100 or less units), then we should be good to go.

Food for thought however, is your insistence that small batch vendors undercut your commercial vendors.  Your rules but, logic dictates from that statement, that by forcing small batch vendors to charge less than your commercially licensed vendors, you're intentionally hurts your commercial vendors.

And IF the dispute is not about price, then WHAT?  I haven't sold anywhere near 100 units combined, and when you factor in the cost of shipping, my price ends up being very close.

I've been trying to get to the bottom of what the objection is.  Perhaps a clear picture could be painted.  I have been trying to work things out, but as with other commenters, I'm so confused.

**Reply by frntc** [reply]
Thanks for ignoring my wish to close this thread. I did explain you the reason in private communication on March 9.
